connectors/graphql/src/main/java/io/camunda/connector/graphql/auth/Authentication.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.graphql.auth;
+
+import com.google.api.client.http.HttpHeaders;
+import com.google.common.base.Objects;
+
+public abstract class Authentication {
```

Comment:
Many of the data classes are identical here and in REST connector. Should we move the common data classes into a separate module, e.g. `connectors-common-library`?